tools/build_rules/prelude_bazel

```diff
@@ -0,0 +1,11 @@
+load(
+    "@io_bazel_rules_pex//pex:pex_rules.bzl",
+    "pex_binary",
+    "pex_library",
+    "pex_test",
+    "pex_pytest",
+)
+load(
+    "@bazel_tools//tools/build_defs/pkg:pkg.bzl",
```

Comment:
Is @bazel_tools defined globally somewhere and is this the proper way to reference the pkg targets we need? I see where @io_bazel_rules_pex is defined, but not @bazel_tools.